recipes/pipfile/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "pipfile" %}
+{% set version = "0.0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash = "f7d9f15de8b660986557eb3cc5391aa1a16207ac41bc378d03f414762d36c984" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  {{ hash_type }}: {{ hash }}
+
+build:
+  number: 0
+  noarch: python
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+
+  run:
+    - python
+    - toml
+
+test:
+  imports:
+    - pipfile
```

Comment:
Could you also add an import test for the api?

Reply:
Ok, I added a import for pipfile.api.